languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -32502,6 +32502,24 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
     <!-- Groß- und Kleinschreibung -->
     <!-- ====================================================================== -->
     <category id="CASING" name="Groß-/Kleinschreibung">
+
```

Comment:
die regel bezieht sich nur auf bspw. was wäre mit anderen Abkürzungen? und brauchen wir überhaupt noch eine Regel, wenn bspw. dann erkannt wird?

Reply:
Ja, es sollte jetzt (großgeschrieben) auch erkannt werden, aber würde dann ja nicht mehr angemarkert werden, wenn es im Satz jetzt großgeschrieben ist, oder? Daher die Regel. 
Kleingeschrieben hat er es schon vorher erkannt und andere Abkürzungen erkennt LT auch am Satzanfang: 
![image](https://user-images.githubusercontent.com/115984740/211768759-10e01cc0-3aec-4a9c-94a1-ed7fdec5978f.png)

Reply:
In  diesem Satz: "Ein Team lebt Bspw. seine Kultur."    wird erkannt, dass "bspw." klein gehört.

Reply:
Wenn ich 'Ein Team lebt bspw. seine Kultur.' teste, ist aber 'bspw' auch nicht im Wörterbuch. Normalerweise sollte durch Hinzufügen von 'bspw' ins Wörterbuch, 'Bspw' am Satzanfang dann nicht mehr als Fehler erkannt werden

Reply:
Ich she gerade, dass 'bspw.' mit Punkt eingetragen ist. Wahrscheinlich werden daraus zwei Token gemacht. Ist das das Problem?